gitea/apiobject.py

```diff
@@ -853,3 +913,13 @@ def convert_time(time: str) -> datetime:
             return datetime.strptime(time[:-3] + "00", "%Y-%m-%dT%H:%M:%S%z")
         except ValueError:
             return datetime.strptime(time[:-3] + "00", "%Y-%m-%dT%H:%M:%S")
+
+class MigrationServices:
+    GIT = "1"
+    GITHUB = "2"
+    GITEA = "3"
+    GITLAB = "4"
+    GOGS = "5"
+    ONEDEV = "6"
+    BITBUCKET = "7"
+    CODEBASE = "8"
```

Comment:
Where is this from? The api, https://try.gitea.io/api/swagger#/repository/repoMigrate, only mentions git, github, gitea and gitlab as string options.

Reply:
Have you tried using other values than the `GIT` one? By the look of the code it works by chance as the default will be `git` unless string matching of the other options listed here:
https://github.com/go-gitea/gitea/blob/a35749893b91db48310d91ae0a32fee3ad3bb901/services/convert/utils.go#L25-L40

Reply:
I've tested github (my main usage) and gitea (the test imports one repository from gitea).

I am using this to take the ids.
https://github.com/go-gitea/gitea/blob/574d8fe6d6675c8aa05e2b75fdbc01c009efd8be/modules/structs/repo.go#L270-L280